admin_guide/disabling_features.adoc

```diff
@@ -0,0 +1,287 @@
+[[admin-guide-disable-feature]]
+= Disabling Features using Feature Gates
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+
+
+As an administrator, you can turn off specific features to specific nodes or the entire platform
+using _feature gates_.
+
+For example, you could turn off new features for production clusters 
+while leaving the features on for test clusters where you can 
+fully test them. Or, prevent access to alpha, beta, or Technology Preview features. 
+
+If you disable a feature that appears in the web console, you might see that feature, but
+no objects are listed. For example, if you disable builds, you can see the Builds tab in the web console. 
+but there are no builds present. 
+
+If you attempt to use commands associated with a disabled feature, such as `oc start-build`, {product-title}
+displays a command not found error.
+
+[NOTE]
+====
+If you disable a feature which any of the applications in the cluster relies on, the application might not
+function properly, depending upon the feature disabled and how the application uses that feature.
+====
+
+Feature gates use a `key=value` pair in the master configuration file
+(*_/etc/origin/master/master-config.yaml_*) and the 
+and kubelet configuration files that describe the feature you want to block.
+
+For example, the following code turns off the Huge Pages feature:
+
+[source,yaml]
+----
+kubernetesMasterConfig:
+  apiServerArguments:
+    feature-gates:
+    - HugePages=false
+ <1>
+  ... 
+  apiServerArguments:
+    feature-gates:
+    - HugePages=false <1>
+----
+
+<1> A key/value pair that turns off a feature:
++
+* `true` enables the specified feature
++
+* `false` disables the specified feature
+
+== Disabling and Enabling Features:
+
+To turn off a feature edit the master configuration file, by default *_/etc/origin/master/master-config.yaml_*:
+
+. For the feature you want to turn off, enter: `<feature_name>=false` under `apiServerArguments`, `apiServerArguments`,
+and `controllerArguments`. 
++
+For example:
++
+[source,yaml]
+----
+kubernetesMasterConfig:
+  apiServerArguments:
+    feature-gates:
+    - HugePages=false
+  apiServerArguments:
+    feature-gates:
+  - HugePages=false
+  controllerArguments:
+    feature-gates:
+  - HugePages=false
+----
+
+. Restart the {product-title} master service to apply the changes.
++
+ifdef::openshift-enterprise[]
+----
+# systemctl restart atomic-openshift-master-api atomic-openshift-master-controllers
+----
+endif::[]
+ifdef::openshift-origin[]
+----
+# systemctl restart origin-master-api origin-master-controllers
```

Comment:
Per bug from @geliu2016 'systemctl restart API, controllers' is not be supported in ocp 310.

Suggestions for improvement: 
change to the 'master-restart' command that new design in 310, of course, this command seems not the final solution, anyway, we need to update it follow the design of 310.